superset/views/sql_lab.py

```diff
@@ -156,7 +156,7 @@ class SavedQueryViewApi(SavedQueryView):
         "sql",
         "extra_json",
     ]
-    show_columns = ["label", "db_id", "schema", "description", "sql", "extra_json"]
+    show_columns = ["id", "label", "db_id", "schema", "description", "sql", "extra_json"]
```

Comment:
I think `black` thinks this line is too long, enabling the pre-commit hooks should help prevent this type of things
https://github.com/apache/incubator-superset/blob/master/CONTRIBUTING.md#git-hooks